projects/playwright-runner/package.json

```diff
@@ -5,19 +5,24 @@
   "main": "src/index.ts",
   "type": "module",
   "scripts": {
-    "run": "node --experimental-strip-types ./src/index.ts"
+    "run": "node --experimental-strip-types ./src/index.js",
+    "test": "node --experimental-strip-types --test \"**/*.spec.ts\"",
+    "test:watch": "node --experimental-strip-types --test --watch \"**/*.spec.ts\""
```

Comment:
Argument order is important here, so we must replicate the whole command to put `watch` before the filename glob.